### PR TITLE
Set limit to display the commit history for checking rebase

### PR DESCRIPTION
**Pull request by chaspy**
close: https://github.com/kamontia/qs/issues/65

### CODE REVIEW

**Comment by chaspy** [comment]
I confirmed the following commands.

```
./qs -n 2
WARN[0000] [ 4]	pickup -> pickup  3f532f0 Add file-6
WARN[0000] [ 3]	pickup -> pickup  b7a122c Add file-7
WARN[0000] [ 2]	pickup -> pickup  5694711 Add file-8
WARN[0000] [ 1]	pickup -> squash  fff4bf3 Add file-9
WARN[0000] [ 0]	pickup -> squash  145235c Add file-10
Do you squash the above commits?(y/n)
```

```
 ./qs -n 2..5
WARN[0000] [ 7]	pickup -> pickup  5845aca Add file-3
WARN[0000] [ 6]	pickup -> pickup  beed52e Add file-4
WARN[0000] [ 5]	pickup -> pickup  4f8855f Add file-5
WARN[0000] [ 4]	pickup -> squash  3f532f0 Add file-6
WARN[0000] [ 3]	pickup -> squash  b7a122c Add file-7
WARN[0000] [ 2]	pickup -> squash  5694711 Add file-8
WARN[0000] [ 1]	pickup -> pickup  fff4bf3 Add file-9
WARN[0000] [ 0]	pickup -> pickup  145235c Add file-10
Do you squash the above commits?(y/n)
```

but it can be specified a value greater than the total number of commit 😢 
this should be validated.

```
./qs -n 2..13
WARN[0000] [11]	pickup -> squash  d4322be Initial commit
WARN[0000] [10]	pickup -> squash  9159282 Add file-0
WARN[0000] [ 9]	pickup -> squash  01e3e83 Add file-1
WARN[0000] [ 8]	pickup -> squash  4c9b106 Add file-2
WARN[0000] [ 7]	pickup -> squash  5845aca Add file-3
WARN[0000] [ 6]	pickup -> squash  beed52e Add file-4
WARN[0000] [ 5]	pickup -> squash  4f8855f Add file-5
WARN[0000] [ 4]	pickup -> squash  3f532f0 Add file-6
WARN[0000] [ 3]	pickup -> squash  b7a122c Add file-7
WARN[0000] [ 2]	pickup -> squash  5694711 Add file-8
WARN[0000] [ 1]	pickup -> pickup  fff4bf3 Add file-9
WARN[0000] [ 0]	pickup -> pickup  145235c Add file-10
Do you squash the above commits?(y/n)
n
```

created issue.  #71

**Comment by chaspy** [comment]
@kamontia 
please review and merge

File: main.go [hunk 1]
```diff
@@ -45,7 +45,16 @@ func validate(n string) {
 
 func display_commit_hash_and_message() {
 	/* Display commit hash and message. The [pickup|..] strings is colored */
-	for i := len(commitMsg) - 2; i >= 0; i-- {
+
+	/* set limit to display the history */
+	var limit int
+	if iNum+2 > len(commitMsg)-2 {
```
Comment (chaspy) [inline main.go]: go fmt removed space 🤔

Comment (kamontia) [reply main.go]: Yes. I think we should conform to the golang tool proposal.

**Comment by chaspy** [comment]
resolved conflict.

@kamontia 
please review and merge.

**Comment by kamontia** [comment]
Thank you for your efforts.
I will implement #71.

**Pull request merged**